### DISCUSSION TITLE
Is it possible to use react-devtools standalone ?

### DISCUSSION THREAD

**Discussion by xavierdemoor** [tauri-apps/tauri#11269] [Q&A]
I'm using Tauri v2 (for the first time) with Vite and React and facing a problem. I can't install an extension, React Devtools.

I tried to use [react-devtools standalone](https://github.com/facebook/react/tree/main/packages/react-devtools) like in the documentation but I can't find a way to make it works. The react-devtools is always "Waiting for React to connect...". I don't know how to place the `<script src="http://localhost:8097"></script>` at the very first script tag in my HTML, maybe it's that. I don't know

Is anybody was able to make this thing works ?

**Comment by FabianLars** [comment]
Yes, i've seen a few people do it (including me in early v1 days). Every vite project should have a top level index.html entry file. Adding it to that didn't work?

**Reply by xavierdemoor** [reply]
Yeah, I searched for a solution during 2h but nothing works. I added the `<script src="http://localhost:8097"></script>` to the `index.html`. It's present in the Tauri's window html, but it doesn't connect with react-devtools. I have nothing bad in the console, like cors problem or anything else.
